.ci/azure/linux_debian.yml

```diff
@@ -225,7 +225,10 @@ jobs:
 
     # Skip test_onnx/test_zoo_models and test_onnx/test_backend due to long execution time
   - script: |
+      # TODO (vurusovs): revert skip of test_core.py::test_register_plugin*,
```

Comment:
Please create a ticket on this

Can we be sure that we won't have another issues in other places?

Reply:
Done - #101968

We prohibit only searching in libopenvino.so folder, other behavior is aligned with previous implementation